app/src/main/java/com/hammer/talkbbokki/presentation/detail/DetailScreen.kt

```diff
@@ -200,7 +224,7 @@ fun FrontCardFace(id: Int) {
 
 @Composable
 fun BackCardFace(
-    item: TopicItem, onClickBookmark: (Boolean) -> Unit
+    item: TopicItem, starter: String, onClickBookmark: (Boolean) -> Unit
```

Comment:
🚫 **[ktlint]** <parameter-list-wrapping> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Parameter should be on a separate line (unless all parameters can fit a single line)